portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsEntries.testcase

```diff
@@ -593,6 +695,110 @@ definition {
 			key_sumValue = "16");
 	}
 
+	@description = "LPS-61197 - Verify that with Single Approver Workflow enabled, the Date Field is properly displayed on an Entry submitted from the Forms Widget"
+	@priority = "5"
+	test FromWidgetWithWorkflowEnabledDisplaysDateField {
+		Navigator.openURL();
+
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			panel = "Site Administration",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName();
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Date");
+
+		Form.save();
+
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			panel = "Site Administration",
+			portlet = "Forms");
+
+		LexiconEntry.gotoEntryMenuItem(
+			menuItem = "Edit",
+			rowEntry = "Created Form Name");
+
+		FormsAdminNavigator.gotoSettings();
+
+		Form.configureWorkflow(workflowDefinition = "Single Approver");
+
+		Form.save();
+
+		ProductMenu.gotoPortlet(
+			category = "Site Builder",
+			panel = "Site Administration",
+			portlet = "Pages");
+
+		SitePages.addPublicPage(pageName = "Form Page");
+
+		Navigator.gotoPage(pageName = "Form Page");
+
+		Portlet.addPG(
+			pageName = "Form Page",
+			portletName = "Form");
+
+		FormPortletNavigator.gotoSelectForm();
+
+		FormPortletConfiguration.selectForm();
+
+		PortletEntry.save();
+
+		Navigator.gotoPage(pageName = "Form Page");
+
+		FormFields.viewDateLabel(
+			fieldLabel = "Date",
+			fieldName = "Date");
+
+		FormFields.editDate(
+			fieldName = "Date",
+			fieldValue = "02/01/2016");
+
+		Button.clickSubmitForPublication();
+
+		Alert.viewSuccessMessage();
```

Comment:
@leticiachiu, can we replace by `FormPortlet.submitSuccessfully();` only?

Reply:
is not possible in this scenario, because this submit isn't on the published form, so we get this error:
`java.lang.Exception: Expected text "Submit" does not match actual text (case insensitive) "Submit for Publication" at "//div[@id='main-content']//button[@type='submit']"`